src/Aspire.Hosting/ApplicationModel/ExpressionResolver.cs

```diff
@@ -154,16 +154,14 @@ async Task<ResolvedValue> EvalValueProvider(IValueProvider vp)
     /// </summary>
     async ValueTask<ResolvedValue> ResolveInternalAsync(object? value)
     {
-        return (value, sourceIsContainer) switch
```

Comment:
Does the summary comment need to be updated at all?

Reply:
It does, thanks. Updated